x/ccv/provider/types/keys.go

```diff
@@ -301,6 +307,17 @@ func getKeyPrefixes() map[string]byte {
 		// consumer validators addresses that need to be pruned.
 		ConsumerAddrsToPruneV2KeyName: 41,
 
+		// LastProviderConsensusValsPrefix is the byte prefix for storing the last validator set
```

Comment:
The comment should start with `LastProviderConsensusValsKey`. 
I'm not sure if the comment should be `LastProviderConsensusValsKeyName` or `LastProviderConsensusValsKey` actually. It seems all key names remove the `Name` suffix from the comment (?)